devito/operator/symbols.py

```diff
@@ -10,16 +10,15 @@ class SymbolRegistry(object):
 
     _symbol_prefix = 'r'
 
-    def __init__(self, nthreads=None, nthreads_nested=None, nthreads_nonaffine=None,
-                 threadid=None):
+    def __init__(self):
         # {name -> generator()} -- to create unique names for symbols, functions, ...
         self.counters = {}
 
         # Special symbols
-        self.nthreads = nthreads
-        self.nthreads_nested = nthreads_nested
-        self.nthreads_nonaffine = nthreads_nonaffine
-        self.threadid = threadid
+        self.nthreads = NThreads(aliases='nthreads0')
+        self.nthreads_nested = NThreadsNested(aliases='nthreads1')
+        self.nthreads_nonaffine = NThreadsNonaffine(aliases='nthreads2')
```

Comment:
More explicit aliases names?

Reply:
I was actually thinking of dropping them since they're useless. The true names are what appears in the generated code and what people typically use in `op.apply(...)`

Reply:
Then would drop aliases yeah.